test_runner/fixtures/neon_fixtures.py

```diff
@@ -28,6 +28,7 @@
 import psycopg2
 import pytest
 import requests
+import toml
```

Comment:
I understood that we use strings because of python toml libraries not always working. I cannot remember any particular case, but seems you are now using it in all of the places I remember.

Reply:
The funny thing is that the `tomllib` which is a part of Python stdlib since 3.11, doesn't support writing toml files, only reading and parsing them.